scripts/microbe_stage/microbe_editor/microbe_editor.as

```diff
@@ -282,9 +282,9 @@ class MicrobeEditor{
                 auto node = hudSystem.world.GetComponent_RenderNode(organelleModel);
                 node.Node.setPosition(cartesianPosition +
                     organelle.organelle.calculateCenterOffset());
-                node.Node.setOrientation(bs::Quaternion(bs::Degree(180),
-                            bs::Vector3(0, 1, 0)) *
-                    bs::Quaternion(bs::Degree(organelle.rotation), bs::Vector3(0, 0, 1)));
+            node.Node.setOrientation(bs::Quaternion(bs::Degree(180),
```

Comment:
I think the code indentation level is broken here.

Reply:
fixed